Source/WebCore/inspector/InspectorThreadableLoaderClient.h

```diff
@@ -39,6 +39,8 @@ class ThreadableLoader;
 
 namespace Inspector {
 
+DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(Loader);
```

Comment:
Use the actual class name.
~~~
DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(InspectorThreadableLoaderClient);
~~~